src/test/java/core/basesyntax/converter/StringTransactionConverterImplTest.java

```diff
@@ -0,0 +1,155 @@
+package core.basesyntax.converter;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.model.Transaction;
+import java.util.ArrayList;
+import java.util.List;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class StringTransactionConverterImplTest {
+    private List<String> stringTransactions;
```

Comment:
does this list really represent transactions? it's likely to be a plain data

Reply:
Sure) it represents List of Strings with information about transactions that we get from Files.readAllLines(Path.of(fileName));
Here i test actual converter that converts Strings to Transaction objects.